Exercise_4/css/partials/side-menu/_small-screen.scss

```diff
@@ -0,0 +1,82 @@
+@import '../variables';
+
+/*SIDE-MENU*/
+
+@media only screen and (max-width: $small-screen_max-width) {
+
+  .side-menu {
+    cursor: pointer;
+    display: flex;
+    font-size: 2rem;
+    margin: 0 2rem;
+    width: 10vw;
+
+    &__content-wrapper {
+      display: none;
+      padding: 1rem;
+    }
+
+    &:focus {
+      .side-menu__content-wrapper {
+        background-color: white;
+        box-shadow: 1rem 2rem 3rem darkgrey;
+        display: flex;
+        flex-direction: column;
+        height: 100vh;
+        left: 0;
+        min-width: 20rem;
+        overflow-y: auto;
+        position: fixed;
+        top: 0;
+        width: 30vw;
+        z-index: 1;
+      }
+    }
+
+    &__search-bar {
+      background-color: white;
+      box-sizing: border-box;
+      font-family: FontAwesome;
+      font-size: 2rem;
+      margin: 0.2rem 0.2rem 0 1rem;
+      padding: 0.2rem;
+      text-align: end;
+    }
+
+    &__footer {
+      align-items: center;
+      display: flex;
+      justify-content: center;
+      margin: 1rem 0;
+    }
+  }
+
+  .vertical-list {
+    list-style-type: none;
+    padding: 0;
+
+    &__item {
+      border-bottom: 0.1rem solid lightgrey;
+      font-size: 1.5rem;
+      margin: 0 1rem;
+      padding: 1rem;
+    }
+
+    &__title {
+      font-size: 2rem;
+      font-weight: bold;
+      margin-left: 1rem;
+    }
+
+    &:last-of-type &-item {
```

Comment:
It was `.list:last-of-type .list-item` and now it is `.vertical-list:last-of-type .vertical-list-item`. Please fix it.

Reply:
@olvnikon I think the mistake was in the previous commit and right now it is good, but maybe you mean that I should use nesting here ??